<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLLForm</class>
 <widget class="QWidget" name="DLLForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="componentsStack">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="componentDXVK">
    <attribute name="title">
     <string>DXVK</string>
    </attribute>
    <widget class="KEditListWidget" name="kcfg_listDXVK">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>491</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="KTitleWidget" name="dxvkListTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>481</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>DXVK Components</string>
     </property>
     <property name="comment">
      <string>Configure DXVK DLLs</string>
     </property>
     <property name="icon">
      <iconset theme="graphics">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="componentDXVKNVAPI">
    <attribute name="title">
     <string>DXVK NVAPI</string>
    </attribute>
    <widget class="KTitleWidget" name="dxvkNVAPIListTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>511</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>DXVK-NVAPI Components</string>
     </property>
     <property name="comment">
      <string>Configure DXVK-NVAPI DLLs</string>
     </property>
     <property name="icon">
      <iconset theme="nvidia">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="KEditListWidget" name="kcfg_listDXVKNVAPI">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>501</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="componentLatencyFleX">
    <attribute name="title">
     <string>LatencyFleX</string>
    </attribute>
    <widget class="KEditListWidget" name="kcfg_listLatencyFlex">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>511</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="KTitleWidget" name="latencFlexListTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>511</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>LatencyFleX Components</string>
     </property>
     <property name="comment">
      <string>Configure LatencyFleX DLLs</string>
     </property>
     <property name="icon">
      <iconset theme="time">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="componentVKD3D">
    <attribute name="title">
     <string>VKD3D</string>
    </attribute>
    <widget class="KEditListWidget" name="kcfg_listVKD3D">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>471</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="KTitleWidget" name="vkd3dListTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>481</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>VKD3D Components</string>
     </property>
     <property name="comment">
      <string>Configure VKD3D DLLs</string>
     </property>
     <property name="icon">
      <iconset theme="graphics">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KEditListWidget</class>
   <extends>QWidget</extends>
   <header>keditlistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KTitleWidget</class>
   <extends>QWidget</extends>
   <header>ktitlewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
